service/create/service.go

```diff
@@ -447,6 +447,7 @@ func (s *Service) runMachine(input runMachineInput) error {
 		return microerror.MaskAny(err)
 	}
 
+	az := input.spec.AWS.Region + input.spec.AWS.AZ
```

Comment:
This is odd somehow. How does Amazon call the AZs? I saw http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#using-regions-availability-zones-launching. There seems to be a dropdown. I couldn't find this in the interface while scanning this quickly. What kind of options are there? I would rather go with this and I cannot imagine there is the option `a`. I think it will be the full region/AZ qualifier.